workflows/rnaseq/rnaseq-standard-fastq.wdl

```diff
@@ -46,20 +44,21 @@ workflow rnaseq_standard {
         Array[File] read_two_fastqs
         File stardb_tar_gz
         String strandedness = ""
-        String output_prefix = "sample"
-        String read_groups = ""
+        String output_prefix
+        String read_groups
```

Comment:
```suggestion
        String? read_groups
```

Reply:
I think `read_groups` should be a required parameter. Even if the user supplies "unknown", at least it would create a BAM with the correct number of read group records.

Reply:
O. I'm not familiar with what happens when STAR isn't supplied that parameter. Does it guess or is the concept of "read groups" lost? I modified the STAR step to just leave out `--outSAMattrRGline` if `read_groups` isn't provided.

Reply:
I suppose that is OK then. I don't like producing BAMs with no @RG records, but maybe allowing that is OK and in practice, we won't do that.